localstack/services/apigateway/invocations.py

```diff
@@ -78,25 +82,32 @@ def is_request_valid(self) -> bool:
         return True
 
     def validate_body(self, resource):
-        # we need a model to validate the body
-        if "requestModels" not in resource or not resource["requestModels"]:
-            return False
+        # if there's no model to validate the body, use the Empty model
+        # https://docs.aws.amazon.com/cdk/api/v1/docs/@aws-cdk_aws-apigateway.EmptyModel.html
+        if not (request_models := resource.get("requestModels")):
+            schema_name = "Empty"
+        else:
+            schema_name = request_models.get(APPLICATION_JSON, "Empty")
 
-        schema_name = resource["requestModels"].get(APPLICATION_JSON)
         try:
             model = self.apigateway_client.get_model(
                 restApiId=self.context.api_id,
                 modelName=schema_name,
             )
         except ClientError as e:
+            # This should not happen, as we validate that we cannot delete a Model if it's used
```

Comment:
I'm always a bit wary of "should not happen" comments - should we simply remove the `try`/`except` handling in this case, if we think it is not a valid code path?

Reply:
Yes, I think it can happen for now until #7872 is merged. I'd be for letting it be there, I'll mark it with a TODO instead, until we're sure this code path can't happen anymore.